### PR TITLE
added header.html

### PR DESCRIPTION
**Pull request by shivampandyajani**
added header.html please add in main

### CODE REVIEW

File: header.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Header</title>
+</head>
+<body>
+  <h1>Header</h1>
```
Comment (the-special-character) [inline header.html]: please use header tag for header feaure

**Review by the-special-character — CHANGES_REQUESTED** [review]
please make chage as per request

File: header.html (hunk 1 above)
Comment (shivampandyajani) [reply header.html]: fix as per your request

Comment (the-special-character) [reply header.html]: good work thank you

**Review by the-special-character — APPROVED** [review]
please make chage as per request

**Pull request merged**